setup.py

```diff
@@ -68,6 +68,7 @@ def run(self):
             objects = compiler.compile(
                 exe.sources,
                 output_dir=self.build_temp,
+                debug=True,
```

Comment:
we were having to deal with a core dumb, and i realized this binary had no symbols.
it's best to have them and not need them than vice versa